include/highfive/h5easy_bits/H5Easy_misc.hpp

```diff
@@ -55,7 +55,7 @@ inline Exception error(const File& file,
     std::ostringstream ss;
     ss << message << std::endl
        << "Path: " << path << std::endl
-       << "Filename: " << file.getName() << std::endl;
+       << "Filename: " << file.getFileName() << std::endl;
```

Comment:
```suggestion
       << "Filename: " << file.getName() << std::endl;
```
See earlier comment about convenience method